section-2/exercises/sviatoslavYurkivSection2.js

```diff
@@ -0,0 +1,44 @@
+
+//Exercise 1:
+const currentTemperature = 0;
+const isRaining = true;
+const cityName = "City";
+
+//Exercise 2: will be an error SyntaxError: Missing initializer in const declaration
+
+// Exercise 3:
+let messageOne = 'Message 1';
+let messageTwo = messageOne;
```

Comment:
In this case you don't need **let** here should be const.  Use let when you know that this variable will be reassigned several times during code execution. 

In exercise 3 you need to create a let variable called  messageOne,  then in separate line assign to this variable some text. 

let messageOne; 
messageOne = 'Message 1';
etc... 

So please fix the code